lib/matplotlib/sankey.py

```diff
@@ -739,7 +740,13 @@ def _get_angle(a, r):
             if label is None or angle is None:
                 label = ''
             elif self.unit is not None:
-                quantity = self.format % abs(number) + self.unit
+                if isinstance(self.format, str):
+                    quantity = self.format % abs(number) + self.unit
+                elif callable(self.format):
+                    quantity = self.format(abs(number))
```

Comment:
Should `abs(number)` be handled as part of callable itself?

Reply:
Great suggestion or catch. This makes sense, `abs()` is removed.